common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -172,46 +174,60 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
-                    {`${hit.annotations.length} ${
-                      hit.annotations.length === 1 ? 'instance' : 'instances'
-                    } ${index &&
-                      `found on image ${index + 1} / ${canvases &&
-                        canvases.length}`} ${
-                      matchingCanvas && matchingCanvas.label.trim() !== '-'
-                        ? ` (page ${matchingCanvas.label})`
-                        : ''
-                    }`}
-                  </HitData>
-                  ...{hit.before}
-                  {/* Use the resource.chars to display the matches individually, rather than hit.match which groups them as a single string */}
-                  {matchingResources.map((resource, i) => (
-                    <span key={i}>
-                      <span
-                        style={{
-                          background: '#944aa0',
-                          color: 'white',
-                        }}
-                      >
-                        {resource?.resource?.chars}
+                  <ListLink
+                    href="/"
+                    style={{ textDecoration: 'none', cursor: 'pointer' }}
+                    onClick={e => {
+                      e.preventDefault();
+                      if (index) {
+                        setIsMobileSidebarActive(false);
+                        setActiveIndex(index || 0);
+                        mainViewerRef &&
+                          mainViewerRef.current &&
+                          mainViewerRef.current.scrollToItem(
+                            index || 0,
+                            'start'
+                          );
+                      }
+                    }}
+                  >
+                    <HitData v={{ size: 's', properties: ['margin-bottom'] }}>
+                      {`${hit.annotations.length} ${
+                        hit.annotations.length === 1 ? 'instance' : 'instances'
+                      } ${index &&
+                        `found on image ${index + 1} / ${canvases &&
+                          canvases.length}`} ${
+                        matchingCanvas && matchingCanvas.label.trim() !== '-'
+                          ? ` (page ${matchingCanvas.label})`
+                          : ''
+                      }`}
+                    </HitData>
+                    <span role="presentation">...{hit.before}</span>
+                    {/* Use the resource.chars to display the matches individually, rather than hit.match which groups them as a single string */}
+                    {matchingResources.map((resource, i) => (
+                      <span key={i}>
+                        <span
+                          style={{
+                            background: '#944aa0',
+                            color: 'white',
+                          }}
+                        >
+                          {resource?.resource?.chars}
```

Comment:
Should we filter out resources that don't null check otherwise we'll have empty DOM being created